subsys/bluetooth/audio/tbs.c

```diff
@@ -793,8 +819,8 @@ static void notify_handler_cb(struct bt_conn *conn, void *data)
 	if (flags->bearer_uri_schemes_supported_list_changed) {
 		LOG_DBG("Notifying Bearer URI schemes supported list: %s", inst->uri_scheme_list);
 
-		err = notify(conn, BT_UUID_TBS_URI_LIST, inst->attrs, &inst->uri_scheme_list,
-			     strlen(inst->uri_scheme_list));
+		net_buf_put_uri_scheme_list(inst, &read_buf);
```

Comment:
`read_uri_scheme_list` also needs to be updated to use `net_buf_put_uri_scheme_list` so we have the same behavior for notification as well as read requests